templates/mentorship_relation_accepted.html

```diff
@@ -1,11 +1,11 @@
-<p>Hi {{ request_sender }},</p>
+<h3 style="color:#8d2165;">Hi {{ request_sender }},</h3>
 <br>
 <p>Congratulations!</p>
 <br>
-<p>Your mentorship request has been accepted by {{ request_receiver }},
-    to be your {{ role }} until {{ end_date }}.
-    Wish you a great mentorship relation with {{ request_receiver }}.</p>
+<p><span style="color: #005c88;">Your mentorship request has been accepted by <strong>{{ request_receiver }}</strong>
+    to be your <span style="color: #005c88;"><strong>{{ role }}</strong></span> until <span style="color: #005c88;"><strong> {{ end_date }}</strong></span>.
+    <span style="color: #005c88;">Wish you a great mentorship relation with <strong>{{ request_receiver }}</strong></span>.</p>    
 <br>
 <p><i>In Systerhood,</i></p>
-<p>Systers Open Source</p>
-<p><a href="http://systers.io/">http://systers.io/</a></p>
+<p style="color:#8d2165;"><strong>Systers Open Source</strong></p>
```

Comment:
Just want to point out that line 10 and 11 need to be replaced with AnitaB.Org Open Source information since its no longer named Systers Open Source. This is also consistent with PR #798.

Reply:
Yaa, just noticed it now Thanks @mtreacy002 for pointing this out. Meanwhile I think the links will also have to be changed.

Reply:
Hey @mtreacy002 Should I Change the signature in all 3 templates with AnitaB instead of Systers?